mission_control/navigator_launch/launch/master_shore.launch

```diff
@@ -1,3 +1,9 @@
 <launch>
```

Comment:
Like the ones one lines 3-5. 

For example, Sub8's perception [launch](Sub8/command/sub8_launch/launch/subsystems/perception.launch), which has tons of parameters, looks very clean.